src/core/modules/impl/handler/AbstractContractHandler.ts

```diff
@@ -36,17 +36,23 @@ export abstract class AbstractContractHandler<State> implements HandlerApi<State
   protected assignWrite(executionContext: ExecutionContext<State>, currentTx: CurrentTx[]) {
     this.swGlobal.contracts.write = async <Input = unknown>(
       contractTxId: string,
-      input: Input
+      input: Input,
+      throwOnError?: boolean
```

Comment:
@jakub-wojciechowski  - this is a new parameter that allow us to set the throwing behaviour independently from the evaluationOptions

Reply:
Great!